Cargo.toml

```diff
@@ -9,6 +9,9 @@ members = [
     "contracts/transfer",
 
     "macros/code-hasher",
+    
+    "prover",
```

Comment:
If we're calling the crate `rusk-prover` isn't it the convention that it should also be in a directory called `rusk-prover`?

Reply:
You're right.
My intention was to call it just `prover`, since it was supposed to not use any `rusk-` dependency.

Then I realized that `rusk-profile` was required, and I miss to update folder properly